README.md

```diff
@@ -104,15 +104,17 @@ Note that this started just as a [fun weekend project](https://localai.io/#backs
 
 Check out the [Getting started](https://localai.io/basics/getting_started/index.html) section in our documentation.
 
-### 💡 Example: Use GPT4ALL-J model
+### 💡 Example: Use Luna AI Llama2 model
 
-See the [documentation](https://localai.io/basics/getting_started/#example-use-gpt4all-j-model-with-docker-compose)
```

Comment:
I'd rather suggest to have a quick example without any specific setup needed ( just copy the model in the folder and done ), from there we can link then to the how-to's

Reply:
Alright how would you like that done?

Reply:
point to the all in one demo?